tests/main_test.go

```diff
@@ -97,13 +99,31 @@ func TestMain(m *testing.M) {
 			}
 			return ctx, nil
 		},
-	).Finish(
+	).AfterEachTest(
+		// Add basic info for test pass/fail for Sonobuoy output.
+		func(ctx context.Context, config *envconf.Config, t *testing.T) (context.Context, error) {
+			result:="passed"
+			if t.Skipped(){
+				result="skipped"
+			}else if t.Failed() {
+				result="failed"
+			}
+			sonobuoyResultsWriter.AddTest(t.Name(), result,nil, "")
+			progressReporter.StopTest(t.Name(),t.Failed(),t.Skipped(),nil)
```

Comment:
NIT: probably need some formatting, needs spaces between operators / etc

Reply:
I'm going to have to figure out whats going on with my IDE (goland). Sometimes it keeps running the gofmt even as I type and its frustrating. Now its not even running it for this project. Not sure why.